### PR TITLE
Upgrade API Calls To Version 0.2 #85

### LINKED ISSUE #85: DOI API v0.2 Compatibility
The DOI API recently transitioned from v0.1 to v0.2. The new version changes some semantics of the endpoints to avoid the use of LIDVIDs as part of the endpoint itself. LIDVIDs should now be provided as query parameters.

The UI needs to be updated to target the v0.2 endpoints in order to work with the latest version of the DOI service.

The Openapi spec for v0.2 of the API can be found here: https://app.swaggerhub.com/apis/PDS_APIs/pds_doi_api/0.2

### PR DESCRIPTION
**Pull request by eddiesarevalo**
-Upgraded the api calls from version 0.1 to 0.2.
-Changed lidvid references to identifier.

Fixes #85

### CODE REVIEW

File: src/sagas/index.js [hunk 1]
```diff
@@ -266,21 +266,31 @@ function* sendSaveReleaseRequest(){
...
-    const identifier = action.payload ? action.payload : '*';
+    let identifier = action.payload ? action.payload : '*';
     let endpoint = Config.api.searchUrl;
-    
+    let isSingleResult = false;
+
     if (identifier.startsWith('10.')) {
         endpoint += '?doi=' + encodeURIComponent(identifier);
-    } else {
-        let searchIdentifier = identifier.replace(/\//g, '-') + '*';
```
Comment (collinss-jpl) [inline src/sagas/index.js:275]: 👍

**Review by collinss-jpl — APPROVED** [review]
Looks good to me, hope it wasn't too difficult to transition to 🙇

**Comment by eddiesarevalo** [comment]
Breaking changes have occurred since the update to the test server. Will commit more fixes before attempting another pull request.

**Review by tloubrieu-jpl — APPROVED** [review]
Thanks @eddiesarevalo , I will merge the pull request, that works. I've created some bugs aside, we'll see there priority with @jordanpadams

**Pull request merged**